shared-module/audiocore/RawSample.c

```diff
@@ -49,26 +55,43 @@ uint8_t common_hal_audioio_rawsample_get_channel_count(audioio_rawsample_obj_t *
 void audioio_rawsample_reset_buffer(audioio_rawsample_obj_t *self,
     bool single_channel_output,
     uint8_t channel) {
+
+    // self->buffer_index = 0;
```

Comment:
Need this uncommented? Otherwise, you can remove it.

Reply:
Done